java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/FieldType.java

```diff
@@ -124,6 +124,7 @@ public enum FieldType implements JsonEnum {
 
 	DenseVector("dense_vector"),
```

Comment:
@maltehedderich please correct me if I am wrong, the `knn_vector` replaces `dense_vector`, right? Should we drop `dense_vector` if that is the case?

Reply:
The only (very weak) reason I would see is compatibility with elasticsearch (and maybe opendistro), where dense vector is a valid field type.

Reply:
Should I drop it within this PR or create another?

Reply:
We could drop it in this pull request I think, thank you.